.kokoro/build.sh

```diff
@@ -54,6 +55,8 @@ export RUNNING_SPANNER_BACKEND_TESTS=1
 export USE_SPANNER_EMULATOR=0
 
 pip3 install .
+pip3 install -e 'git+https://github.com/q-logic/python-spanner.git@django_test_fixes#egg=google-cloud-spanner'
+pip3 install -e 'git+https://github.com/q-logic/python-spanner-django.git@dj_tests_against_emulator#egg=django-google-spanner'
```

Comment:
Do you still need to install from this branch? I see it still has `add_dummy_where`?

Reply:
We can safely drop only the first install line, and I've already did it.